sal_interfaces/MTDome/MTDome_Commands.xml

```diff
@@ -80,42 +86,42 @@
   <SALCommand>
     <Subsystem>MTDome</Subsystem>
     <EFDB_Topic>MTDome_command_stopLouvers</EFDB_Topic>
-    <Description>Stop all louvers.</Description>
+    <Description>Stop all louvers. This should also be used for an emergency stop.</Description>
   </SALCommand>
   <SALCommand>
     <Subsystem>MTDome</Subsystem>
     <EFDB_Topic>MTDome_command_closeShutter</EFDB_Topic>
-    <Description>Close the shutter.</Description>
+    <Description>Close the shutter. This should also be used for an emergency stop.</Description>
   </SALCommand>
   <SALCommand>
     <Subsystem>MTDome</Subsystem>
     <EFDB_Topic>MTDome_command_openShutter</EFDB_Topic>
-    <Description>Open the shutter.</Description>
+    <Description>Open the shutter. This should also be used for an emergency stop.</Description>
```

Comment:
I suggest reverting this change. This operation is not a "stop" and I think it would be a very unusual emergency that required _opening_ the shutter.

Reply:
Yeah, that was a bit too enthusiastic copying and pasting. Reverted.